packages/react-aria-components/src/Select.tsx

```diff
@@ -243,7 +245,8 @@ function SelectValue<T extends object>(props: SelectValueProps<T>, ref: Forwarde
     values: {
       selectedItem: state.selectedItem?.value as T ?? null,
       selectedText: state.selectedItem?.textValue ?? null,
-      isPlaceholder: !selectedItem
+      isPlaceholder: !selectedItem,
+      selectedChildren: rendered
```

Comment:
I wonder if we need this now that we have support for accessing `defaultChildren` from the render prop function (I added that for color picker)?